src/Asset/Installer/AssetsProvider.php

```diff
@@ -48,7 +52,11 @@ public function provideDirectoriesForTheme(ThemeInterface $rootTheme): iterable
     public function provideDirectoriesForBundle(BundleInterface $bundle): iterable
     {
         yield $bundle->getPath() . '/Resources/public' => '/bundles/' . $this->getPublicBundleName($bundle);
-        yield $bundle->getPath() . '/public' => '/bundles/' . $this->getPublicBundleName($bundle);
+    }
+
+    public function provideDirectoriesForThemeBundle(BundleInterface $bundle): iterable
+    {
+        yield $bundle->getPath() . '/Resources/public' => '/bundles/_theme/' . $this->getPublicBundleName($bundle);
```

Comment:
```suggestion
        yield $bundle->getPath() . '/Resources/public' => '/bundles/_themes/' . $this->getPublicBundleName($bundle);
```

To be compatible with PathResolver implementation: `return sprintf('%s/_themes/%s/%s', $prefixPath, $theme->getName(), $relativePath);`